### ISSUE TITLE
Optional fields in PivotalTracker

### ISSUE THREAD

**Issue by paul-mesnilgrente** [DataFire/Integrations#37]
The pivotal api v5 allows you to add some attributes/associations to the response as GET parameters, you can see it [here](https://www.pivotaltracker.com/help/api#Response_Customization) on the section named "Selecting Attributes to Include in Responses"

Currently the PivotalTracker integration doesn't allow you to do it. Is there a way to do it? I'll be glad to work on a pull request for it or whatever. I need a starting point though and be sure that my work will be integrated if it's worth it :)

**Comment by bobby-brennan** [comment]
Yup this should be doable. I think you just need to add a `fields` parameter to the OpenAPI for each operation.

You can look in `./patches` for examples of how to patch the OpenAPI spec. Then you should be able to run `node scripts/apis-guru --name pivotaltracker` to apply the patch.

Hope that helps - if you get stuck feel free to reach out!

**Comment by paul-mesnilgrente** [comment]
Nice, I'll have a look this Friday and see if I can do something about it :) I'll keep you posted.
